### DISCUSSION TITLE
Reveal.js Presentation column padding

### DISCUSSION THREAD

**Discussion by garthtarr** [quarto-dev/quarto-cli#869] [Feature Requests]
I think there should be some padding added to give some space between columns in presentations. At the moment I add a small % "extra" column in between, but is there a better way?

<img width="772" alt="Screen Shot 2022-05-13 at 11 45 25 pm" src="https://user-images.githubusercontent.com/2189134/168297529-822f6c26-3ff4-4e12-b05d-11e8d6cbac10.png">

````r
---
title: "Column padding"
format: revealjs
---

## Column padding needed

:::: {.columns}

::: {.column width="50%"}

:::{.callout-tip}
Testing
:::

```{r, echo=TRUE}
rnorm(1)
```

Text runs into the next column..

:::



::: {.column width="50%"}

:::{.callout-tip}
Testing
:::

```{r, echo=TRUE}
rnorm(1)
```

Text runs into the next column.

:::

::::


## Code blocks

:::: {.columns}

::: {.column width="50%"}

```{r, echo=TRUE}
rnorm(1)
```

:::



::: {.column width="50%"}

```{r, echo=TRUE}
rnorm(1)
```

:::

::::
````

**Comment by jjallaire** [comment]
Good observation! Yes, we do need some padding there. I've added 0.5em of padding here: https://github.com/quarto-dev/quarto-cli/commit/805f7ad84e17a2297657247f34b07e7e2f9b5bac

**Reply by garthtarr** [reply]
Outstanding, thanks! For completeness, he's an updated screenshot of the post-fix behaviour:

<img width="764" alt="Screen Shot 2022-05-14 at 1 33 04 pm" src="https://user-images.githubusercontent.com/2189134/168409317-6e16fe98-3ecd-4e6f-af92-393937f384f9.png">

**Comment by mcanouil** [comment]
Closing this as completed.
